system/system_monitor/src/cpu_monitor/cpu_monitor_base.cpp

```diff
@@ -198,18 +243,24 @@ void CPUMonitorBase::checkUsage(diagnostic_updater::DiagnosticStatusWrapper & st
           } else {
             whole_level = std::max(whole_level, level);
           }
+          if (cpu_name != "all") cpu_usage.cpus.push_back(cpu_status);
         }
       }
     }
   } catch (const std::exception & e) {
     stat.summary(DiagStatus::ERROR, "mpstat exception");
     stat.add("mpstat", e.what());
     std::fill(usage_check_cnt_.begin(), usage_check_cnt_.end(), 0);
+    cpu_usage.all.status = STATUS.STALE;
+    publishCpuUsage(cpu_usage);
```

Comment:
Here, the data of each CPU may be pushed so it needs to clear the cpu_usage.cpus.

Reply:
Add clearing process after publishCpuUsage.

Reply:
I intended to clear it before publishing because the number of CPUs is wrong when an exception.

Reply:
I modified the place.